include/engine/guidance/post_processing.hpp

```diff
@@ -16,6 +17,15 @@ namespace guidance
 // passed as none-reference to modify in-place and move out again
 std::vector<RouteStep> postProcess(std::vector<RouteStep> steps);
 
+// trim initial/final segment of very short length. Function set up for in-place modification
+void trimShortSegments(std::vector<RouteStep> &steps, LegGeometry &geometry);
+
+//assign relative locations to depart/arrive instructions
+void assignRelativeLocations(std::vector<RouteStep> &steps,
+                             const LegGeometry &geometry,
+                             const PhantomNode &source_node,
+                             const PhantomNode &target_node);
+
```

Comment:
Hm the other functions here take a vector by value, modify it and move it out again. Should we be consistent here? 

(Not that big of a deal for sure)